index.js

```diff
@@ -54,39 +56,46 @@ async function deployActions({run, functionsDir, timeout, memory}) {
 module.exports = {
   // Execute before build starts.
   onPreBuild: async ({utils, constants, inputs}) => {
-    try {
-      if (!process.env.NIMBELLA_LOGIN_TOKEN) {
-        utils.build.failBuild(
-          'Nimbella login token not available. Please run `netlify addons:create nimbella` at the base of your local project directory linked to your Netlify site.'
-        );
-      }
+    if (!process.env.NIMBELLA_LOGIN_TOKEN) {
+      utils.build.failBuild(
+        'Nimbella login token not available. Please run `netlify addons:create nimbella` at the base of your local project directory linked to your Netlify site.'
+      );
+    }
 
-      const nimConfig = join(require('os').homedir(), '.nimbella');
-      await utils.cache.restore(nimConfig);
+    await utils.cache.restore(nimConfig);
 
-      const loggedIn = existsSync(nimConfig);
-      // Login if not logged in before.
-      if (loggedIn) {
-        console.log('\nUsing the following namespace.');
+    const loggedIn = existsSync(nimConfig);
+    // Login if not logged in before.
+    if (loggedIn) {
+      console.log('Using the following namespace.');
```

Comment:
Can we print the console message after `nim auth current` succeeds?

Reply:
Updated.